config/samples/flows_v1beta2_flowcollector.yaml

```diff
@@ -137,19 +142,19 @@ spec:
     quickFilters:
     - name: Applications
       filter:
-        flow_layer: 'app'
+        flow_layer: '"app"'
       default: true
     - name: Infrastructure
       filter:
-        flow_layer: 'infra'
+        flow_layer: '"infra"'
     - name: Pods network
       filter:
-        src_kind: 'Pod'
-        dst_kind: 'Pod'
+        src_kind: '"Pod"'
+        dst_kind: '"Pod"'
       default: true
     - name: Services network
       filter:
-        dst_kind: 'Service'
+        dst_kind: '"Service"'
```

Comment:
this is bit ugly and not user friendly why did u change the APIs
https://github.com/netobserv/network-observability-operator/pull/613/files#diff-1aad53ab28e869c35c03e94f19d4e37402b869c34eabfd55317931ab5e2b1436R863 ?

Reply:
cc'd @memodi

Reply:
it's a perf improvement for queries, have quotes means doing an exact match instead of a partial match, which is more performant